docs/configuration/secure.md

```diff
@@ -0,0 +1,614 @@
+        Header set Feature-Policy "sync-xhr 'self'"
+
+        # Working with ILIAS in most use cases:
+        Header set Content-Security-Policy "default-src 'self'; connect-src 'self'; script-src 'self' 'unsafe-inline' 'unsafe-eval' blob:; style-src 'self' 'unsafe-inline' data:; img-src 'self' 'unsafe-inline' data:; font-src 'self' 'unsafe-inline' data:; media-src 'self' 'unsafe-inline' data:"
+
+        # Working with ILIAS in most use cases:
+        Header set X-Content-Security-Policy "default-src 'self'; connect-src 'self'; script-src 'self' 'unsafe-inline' 'unsafe-eval' blob:; style-src 'self' 'unsafe-inline' data:; img-src 'self' 'unsafe-inline' data:; font-src 'self' 'unsafe-inline' data:; media-src 'self' 'unsafe-inline' data:"
+    </IfModule>
+```
+
+ ##### NGINX
+ You can simply add this in your ```server``` configuration:
+
+ ```
+  # Content-Security
+         add_header X-Frame-Options "SAMEORIGIN";
+         add_header X-Content-Type-Options nosniff;
+         add_header X-XSS-Protection "1; mode=block";
+         add_header Referrer-Policy "strict-origin";
+         add_header Feature-Policy "sync-xhr 'self';
+
+         # Working with ILIAS in most use cases:
+         add_header Content-Security-Policy "default-src 'self'; connect-src 'self'; script-src 'self' 'unsafe-inline' 'unsafe-eval' blob:; style-src 'self' 'unsafe-inline' data:; img-src 'self' 'unsafe-inline' data:; font-src 'self' 'unsafe-inline' data:; media-src 'self' 'unsafe-inline' data:";
+         #compatibility to  Microsoft Internet Explorer 10:
+         add_header X-Content-Security-Policy "default-src 'self'; connect-src 'self'; script-src 'self' 'unsafe-inline' 'unsafe-eval' blob:; style-src 'self' 'unsafe-inline' data:; img-src 'self' 'unsafe-inline' data:; font-src 'self' 'unsafe-inline' data:; media-src 'self' 'unsafe-inline' data:";
+ ```
+
+note:  
+If you use a proxied [Chat Server](https://github.com/ILIAS-eLearning/ILIAS/blob/release_5-4/Modules/Chatroom/README.md), you MUST add the url to the CSP definition:  
+```connect-src 'self' wss://onscreenchat.%HOSTNAME% https://onscreenchat.%HOSTNAME%;```
+
+If you use other external content (e.g. for a SCORM module or external video content) you have to add a ```default-src``` (The default-src is the default policy for loading content such as JavaScript, Images, CSS, Fonts, AJAX requests, Frames, HTML5 Media from other sites):
+
+e.g. for https://www.youtube.com as source:
+
+```default-src 	'self' www.youtube.com; ...```
+
+#### Validate the header configuration
+It is recommended to validate your configuration with the services from https://securityheaders.com/.  
+Try to reach **A** grade.
+
+### Enable OCSP stapling (TLS Certificate Status Request)
+
+The  TLS/SSL extension [OCSP stapling](https://en.wikipedia.org/wiki/OCSP_stapling) provides a way to improve the performance (cause the webserver is will do this instead of the browser client) of SSL negotiation while maintaining visitor privacy.
```

Comment:
```suggestion
The  TLS/SSL extension [OCSP stapling](https://en.wikipedia.org/wiki/OCSP_stapling) provides a way to improve the performance (as the webserver will fetch the OCS instead of the browser client) of SSL negotiation while maintaining visitor privacy.
```